app/views/home/temp_dashboard.html.erb

```diff
@@ -0,0 +1,22 @@
+<div id="dashboard">
+  <div id="page">
+    <div class="banner">
+      <iron-icon icon="icons:error" class="alert"></iron-icon>
+ You've been assigned as an author on a new paper. <a href="#">Begin review.</a>
```

Comment:
This should ne `assigned as a reviewer`, I was actually going to the banner out for now. There is no simple way to tell if they are 'new' short of if they haven't commented or writing a lot more code.